docs/en/connector-v2/source/MongoDB.md

```diff
@@ -0,0 +1,73 @@
+# MongoDb
+
+> MongoDb source connector
```

Comment:
Hi, we have recently formulated a new document format specification. You can get details from https://github.com/apache/incubator-seatunnel/pull/2625.

Please modify your document according to the requirements of the new document format.

Thank you.

Reply:
@EricJoy2048 Thank you for your reminder